base-notebook/Dockerfile

```diff
@@ -87,8 +86,7 @@ CMD ["start-notebook.sh"]
 COPY start.sh /usr/local/bin/
 COPY start-notebook.sh /usr/local/bin/
 COPY start-singleuser.sh /usr/local/bin/
-COPY jupyter_notebook_config.py /home/$NB_USER/.jupyter/
-RUN chown -R $NB_USER:users /home/$NB_USER/.jupyter
+COPY jupyter_notebook_config.py /etc/jupyter/
```

Comment:
Should this be chown'ed to the `jovyan` user so that root isn't required to edit it?